server/tinylicious/src/services/inMemorycollection.ts

```diff
@@ -83,12 +83,26 @@ export class Collection<T> implements ICollection<T> {
 		});
 	}
 
+	private removeOneInternal(value: any): void {
```

Comment:
Are these changes related to your PR? I'm not sure what's the relationship, and what these changes do.

Reply:
yes, I will create a new PR for it as these methods are throwing on  my local tests.